src/openrct2-android/app/build.gradle

```diff
@@ -98,7 +98,7 @@ android.applicationVariants.all { variant ->
             into "$variant.mergeAssets.outputDir/data/title"
         }
         download {
-            src 'https://github.com/OpenRCT2/objects/releases/download/v1.0.10/objects.zip'
+            src 'https://github.com/OpenRCT2/objects/releases/download/v1.0.11/objects.zip'
```

Comment:
Why is this in this commit? (51315aa0444ccab58a229a7570ee93f8e69caf1f — weird commit message, too.)

Reply:
Because of a mistake in the second commit. I meant to fix it in the last rebase but forgot about this change.

Reply:
Fixed now